tests/test_models_requires.py

```diff
@@ -1,27 +0,0 @@
-import pytest
-from plette import models
-
-def test_requires_python_version():
```

Comment:
I am wondering if we should have more tests around the Requires model -- noticed this is getting removed.

Reply:
This seems as it is getting removed, because on that branch I split the tests from one file to many small ones.
You can find these tests here:
https://github.com/sarugaku/plette/blob/e3fa787f16ed78766c186cffe32c24f20020c7d6/tests/test_models.py#L69

And yes, I agree with you with need more tests! The current suite is very weak, and I have been working constantly to improve it (first, I did add all the `examples/Pipfile`s and the integration tests around them. This PR also adds another tests for a case which was caught in upstream merging.

I will add more in the near future.